sympy/functions/elementary/trigonometric.py

```diff
@@ -375,7 +375,7 @@ def _eval_rewrite_as_Pow(self, arg):
             return I*x**-I / 2 - I*x**I /2
 
     def _eval_rewrite_as_cos(self, arg):
-        return -cos(arg + S.Pi/2)
+        return -cos(arg + S.Pi/2, evaluate=False)
```

Comment:
There are several ways to introduce the minus sign. This seems to be the original expression, but it could be changed by the same token unless there are good grounds for the use of this form. I'd personally prefer `cos(arg - S.Pi/2)` or `cos(S.Pi/2 - arg)` .